### PR TITLE
bitboxbase: Don't abort on wrong screen orientation

### PR DESCRIPTION
**Pull request by NickeZ**
Also don't rotate screen in bootloader and always set it to not flipped
when rebooting

### CODE REVIEW

File: src/workflow/reboot.c [hunk 1]
```diff
@@ -26,7 +26,11 @@ static void _reboot(void)
         .value = sectrue_u8,
     };
     upside_down_t upside_down = {
+#if PLATFORM_BITBOX02 == 1
         .value = screen_is_upside_down(),
+#elif PLATFORM_BITBOXBASE == 1
+        .value = false,
```
Comment (benma) [inline src/workflow/reboot.c]: `#else #error` 

or `#if PLATFORM_BITBOXBASE == 1 ... #else ... `

Comment (NickeZ) [reply src/workflow/reboot.c]: ack

Comment (benma) [reply src/workflow/reboot.c]: still missing `#else #error`

Comment (conte91) [reply src/workflow/reboot.c]: > or `#if PLATFORM_BITBOXBASE == 1 ... #else ... `

This wins

File: src/workflow/workflow.c [hunk 2]
```diff
@@ -73,11 +74,13 @@ static void _select_orientation_done(bool upside_down)
...
     ui_screen_stack_switch(show_logo);
 }
+#endif
 
 void workflow_start_orientation_screen(void)
 {
 #if PLATFORM_BITBOXBASE == 1
-    _select_orientation_done(false);
+    component_t* show_logo = show_logo_create(workflow_start, 200);
+    ui_screen_stack_switch(show_logo);
```
Comment (conte91) [inline src/workflow/workflow.c]: Direct call to `_select_orientation_done(false)` instead? So we don't have to track future changes in the latter.

File: src/workflow/reboot.c (hunk 1 above)
Comment (NickeZ) [reply src/workflow/reboot.c]: I think I added that, must have force pushed over it

File: src/workflow/workflow.c (hunk 2 above)
Comment (NickeZ) [reply src/workflow/workflow.c]: Yeah, it's either that and doing ifdefs around "upside_down" or copying these two lines. I did the second. I think the whole "orientation screen" doesn't make sense from the BBB point of view.

File: src/ui/oled/oled.c [hunk 3]
```diff
@@ -253,7 +254,8 @@ void oled_mirror(bool mirror)
     }
 #elif PLATFORM_BITBOXBASE == 1
     (void)mirror;
-    Abort("BitBoxBase cannot mirror screen");
+    // DON'T ABORT HERE, this is called in the bootloader
+    screen_print_debug("BitBoxBase cannot mirror screen", 5000);
```
Comment (benma) [inline src/ui/oled/oled.c]: `#else #error` also missing here :laughing: but let's merge this so we can get on with it :D

Comment (NickeZ) [reply src/ui/oled/oled.c]: But here it is definitely a compile error because "mirror" would be unused!

**Pull request merged**